pages/guestbook.js

```diff
@@ -1,4 +1,4 @@
-import redis from '@/lib/redis';
+import db from '@/lib/planetscale';
```

Comment:
This import is causing the `Module not found: Error: Can't resolve 'tls'` error during `yarn build` because its call site below is commented out. Using an imported module inside `getStaticProps` or `getServerSideProps` is the signal to Next that it should be compiled only into the server bundle and not into the browser bundle. So right now it's trying to compile Node.js built-in modules, like `tls`, into a browser bundle and failing.

Uncommenting the `db.query` call below should fix this build error.